CIScripts/DockerFiles/iis-tcptest/Dockerfile

```diff
@@ -1,10 +0,0 @@
-FROM microsoft/iis:windowsservercore-10.0.14393.1358
-
-RUN mkdir C:\TCPTest
-RUN echo "<h1>Hello, Contrail!</h1>" > C:\TCPTest\index.html
```

Comment:
you shouldn't have removed this, the default SimpleHTTPServer page will be too big